### DISCUSSION TITLE
Generate vitals every 4 hours when inpatient or snf

### DISCUSSION THREAD

**Discussion by MichelleFekula** [synthetichealth/synthea#1366] [Ideas]
How can the modules be modified to return vitals every 4 hours during Inpatient and SNF?

**Comment by jawalonoski** [comment]
This can be done, but it isn't quick or easy.

There is no setting that controls frequency of vital signs during a stay.

The very high-level way this could be accomplished: each module that has an inpatient or SNF stay of interest would have to be modified (using the Module Builder) to introduce a loop (or shorten an existing loop) to take vitals at the desired rate (e.g., 4 hours).

**Comment by MichelleFekula** [comment]
Thank for the information.  Is there an example in any of the modules for me to be able to follow how to setup the loop in the module builder?
